.gitignore

```diff
@@ -680,6 +680,7 @@ release
 releases/*
 keys/*
 .hemtt/local
+hemtt
```

Comment:
Undo this

Reply:
Hasn't been updated? Still needs changing

Reply:
think I missed the commit but should be sorted now